src/Host/Client/Impl/BrokerServices/RemoteUriWebService.cs

```diff
@@ -22,17 +23,18 @@ public class RemoteUriWebService : IRemoteUriWebService {
 
         private Uri PostUri { get; }
 
-        public async Task GetResponseAsync(HttpListenerContext context, string localBaseUrl, string remoteBaseUrl, CancellationToken ct) {
+        public async Task GetResponseAsync(HttpListenerContext context, string localBaseUrl, string remoteBaseUrl, string webSocketsScheme, CancellationToken ct) {
             string postUri = null;
             if (context.Request.IsWebSocketRequest) {
-                UriBuilder ub = new UriBuilder(PostUri) { Scheme = "ws" };
+                UriBuilder ub = new UriBuilder(PostUri) { Scheme = webSocketsScheme };
```

Comment:
The PostUri points to an endpoint on the broker `http[s]://<broker>:<port>/remoteuri`. This should match whatever scheme that is used to connect with the broker.